fleet_adapter_mirfm/test/test_flake8.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ament_flake8.main import main_with_errors
+import pytest
+
+
+@pytest.mark.flake8
+@pytest.mark.linter
+def test_flake8():
```

Comment:
Are these tests actually being used? If so, github actions should be added to run them, else they should be removed.

Reply:
You're right they were automatically generated, removed in [520d2ca](https://github.com/osrf/fleet_adapter_mir/pull/27/commits/520d2cab116e91ee03871b0263157cc94e0e5b33)